tool/grind.dart

```diff
@@ -89,6 +90,28 @@ vulcanize(String filepath) {
     'main.dart.js',
     '--exclude',
     'packages/codemirror/codemirror.js',
+    '--exclude',
+    'embed_components.html',
+    '--exclude',
+    'animated-dropdown.html',
+    filepath
+  ], workingDirectory: 'build/web');
+  if (result.exitCode != 0) {
+    fail('error running vulcanize: ${result.exitCode}\n${result.stderr}');
+  }
+  HtmlFile.asFile.writeAsStringSync(result.stdout);
```

Comment:
Can you change this var reference here and at the definition (`HtmlFile` ==> `htmlFile`)?